### ISSUE TITLE
k8s-driver-manager use a unified version

### ISSUE THREAD

**Issue by lengrongfu** [NVIDIA/gpu-operator#642]
current in the helm chart values.yaml define many k8s-driver-manager image, but version unlikeness, so we can define a unified field and unified version?

**Comment by cdesiniotis** [comment]
Yes you are right, we do have multiple components which have their own `k8s-driver-manager` configuration. This could be unified with some refactoring. If you are interested, contributions are welcome on our Gitlab repository (which is where we do development currently): https://gitlab.com/nvidia/kubernetes/gpu-operator

**Comment by cdesiniotis** [comment]
We have migrated to GitHub, so please raise any PRs here.
